Added users last access to homes

buddysal opened the pull request:
Added users last access date to homes and removed the commented section that was a place holder.

smile0711 commented on api/src/services/member/member.service.ts:
```diff
@@ -189,6 +189,7 @@ export class MemberService {
       lastName: member.lastname,
       chatdefault: member.chatdefault,
       primary_role_id: member.primary_role_id,
+      lastAccess: member.last_activity,
```
Normal users will not need this informations, just anyone with admin right for that house. Can we remove this line?

smile0711 commented on spa/src/components/place/home/main2d.vue:
```diff
@@ -135,9 +136,30 @@ export default Vue.extend({
     this.getData();
+    this.checkAdmin();
+  },
+  watch: {
```
A watch maybe needed but is the better thing to watch for might be this.$route.params.username

buddysal commented:
I updated my PR and fork to the changes you suggested. Hopefully this is more along the lines of how you were talking about doing it. I changed it in a way that hopefully is easier to expand on in future by adding a canStaff area to the member controller, so that can admin and can staff will be their own entity (and prevent possible overlap and wrong access going to wrong things). I also changed the watch to the username instead.
